src/wp-includes/class-wp-scripts.php

```diff
@@ -295,14 +304,34 @@ public function do_item( $handle, $group = false ) {
 		}
 
 		$before_handle = $this->print_inline_script( $handle, 'before', false );
-		$after_handle  = $this->print_inline_script( $handle, 'after', false );
 
 		if ( $before_handle ) {
 			$before_handle = sprintf( "<script%s id='%s-js-before'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), $before_handle );
 		}
 
+		// Eligible loading strategies will only be 'async', 'defer', or ''.
+		$strategy     = $this->get_eligible_loading_strategy( $handle );
+		$after        = ( '' === $strategy ) ? 'after' : 'after-standalone';
```

Comment:
See above:
```suggestion
		$after        = ( 'defer' === $strategy || 'async' === $strategy ) ? 'after' : 'after-standalone';
```

Note my other comment that I don't think we should use this line as a whole, but this comment here is just to clarify my comment right above.

Reply:
@felixarntz this is a good point indeed. I think your suggested change is the incorrect way around though. Right now, we say that if there is no (or default blocking) strategy, print `after` else print `after-standalone`. 

Your proposed changes would print `after` if there was a `defer` or `async` strategy. I think it needs to be:

```
$after        = ( 'defer' === $strategy || 'async' === $strategy ) ? 'after-standalone' : 'after';
```

Correct?

Reply:
@felixarntz If I am reading correctly, you want to remove the line altogether, which I am struggling to make sense of. I agree that we should change it to be explicitly checking for async or defer (as that can be easily updated as new strategies are added) but removing it will have knock on effects. 

Perhaps just me misunderstanding, I'm going to effect the changes that I highlighted above and we can have follow up comms if I was incorrectly understanding this.